crmsh/utils.py

```diff
@@ -3171,4 +3171,15 @@ def ssh_command():
     if config.core.no_ssh:
         raise NoSSHError(constants.NO_SSH_ERROR_MSG)
     return "ssh"
+
+
+def load_cib_file_env():
+    if options.regression_tests or ServiceManager().service_is_active("pacemaker.service"):
+        return
+    logger.warning("Cluster is not running, loading CIB_file from environment variable")
+    cib_file = os.environ.setdefault('CIB_file', constants.CIB_RAW_FILE)
```

Comment:
Not 100% sure, keep in mind the existing use case:
`CIB_file="/path/to/cib.xml" crm configure show`

I wonder the safer code should be like this 

```suggestion
    cib_file = os.getenv('CIB_file')
    if cib_file is None:
        cib_file = os.environ.setdefault('CIB_file', constants.CIB_RAW_FILE)
    logger.warning(f"Cluster is not running, loading the CIB file from {cib_file}")
```

Reply:
https://docs.python.org/3/library/stdtypes.html#dict.setdefault
> If key is in the dictionary, return its value. If not, insert key with a value of default and return default.

So, it's OK for the existing one, just check if the file exists

Reply:
Great! More clean code then! Even so, I still suggest

```suggestion
    cib_file = os.environ.setdefault('CIB_file', constants.CIB_RAW_FILE)
    logger.warning(f"Cluster is not running, loading the CIB file from {cib_file}")
```